packages/next/src/server/app-render/metadata-insertion/create-server-inserted-metadata.tsx

```diff
@@ -34,7 +42,18 @@ export function createServerInsertedMetadata() {
       metadataToFlush = metadataResolver()
       const html = await renderToString({
         renderToReadableStream,
-        element: <>{metadataToFlush}</>,
+        element: (
+          <>
+            {metadataToFlush}
+            <script
+              defer
```

Comment:
why defer? this won't run until the document has completely loaded. But all of the icons we want to hoist are know to be in the document already. this can just be a sync function.